browser_patches/firefox/patches/bootstrap.diff

```diff
@@ -6805,6 +6805,19 @@ index 36da62856100a9db680f2515d9dc5427d8945233..b29d93917386d2f70d35122da0fdbe0a
        ? "https://firefox.settings.services.mozilla.com/v1"
        : gServerURL;
    },
+diff --git a/toolkit/components/startup/nsAppStartup.cpp b/toolkit/components/startup/nsAppStartup.cpp
+index 1fc58612ab8efe2f7cfd5da29026f39c60e49584..9fc00a855678b8ae9e4549cdead7be093a3c3090 100644
+--- a/toolkit/components/startup/nsAppStartup.cpp
++++ b/toolkit/components/startup/nsAppStartup.cpp
+@@ -336,7 +336,7 @@ nsAppStartup::Quit(uint32_t aMode) {
+     nsCOMPtr<nsISimpleEnumerator> windowEnumerator;
+     nsCOMPtr<nsIWindowMediator> mediator(
+         do_GetService(NS_WINDOWMEDIATOR_CONTRACTID));
+-    if (mediator) {
++    if (ferocity == eAttemptQuit && mediator) {
```

Comment:
Should this be `ferocity != eForceQuit` ?

Reply:
It's the same here, I've considered both, since you think ferocity != eForceQuit changing to it :)